app/models/ticket_subscription.rb

```diff
@@ -2,6 +2,15 @@ class TicketSubscription < ApplicationRecord
   belongs_to :ticket
   belongs_to :user
 
+  validates :ticket_id,
+    presence: true
```

Comment:
Do I have only id? I think that this one is better if ticket is necessary.
`validates :ticket, presence: true`